aspnetcore/tutorials/first-mongo-app.md

```diff
@@ -50,10 +50,10 @@ Enable MongoDB and MongoDB Shell access from anywhere on the development machine
 
 1. Download and Install MongoDB Shell:
    * macOS/Linux: Choose a directory to extract the MongoDB Shell to. Add the resulting path for `mongosh` to the `PATH` environment variable.
-   * Windows: MongoDB Shell (mongosh.exe) is installed at *C:\Users\<user>\AppData\Local\Programs\mongosh*. Add the resulting path for `mongosh.exe` to the `PATH` environment variable.
+   * Windows: MongoDB Shell (mongosh.exe) is installed at *C:\\Users\\\<user>\\AppData\\Local\\Programs\\mongosh*. Add the resulting path for `mongosh.exe` to the `PATH` environment variable.
```

Comment:
Adds extra `\` to escape the path backslashes.
This fixes an invalid path in the docs (there should be a backslash between users and <user>):

![image](https://github.com/user-attachments/assets/6a531d8c-edac-4b8a-8aae-b4d8c4b226f1)